romancal/datamodels/__init__.py

```diff
@@ -22,4 +22,6 @@
 
            "ImageModel",
 
+           "RampModel", "GLS_RampFitModel", "RampFitOutputModel",
```

Comment:
These need to be imported from the respective modules in order to be able to define them here.
I'll fix it in #115